spk/rutorrent/src/rtorrent.rc

```diff
@@ -1,9 +1,9 @@
 scgi_port = localhost:8050
 directory = @download_dir@
-session = /usr/local/rutorrent/var/.session
+session = @var_dir@/.session
```

Comment:
my preferred way would be to use
`/var/packages/rutorrent/var`
this is availabel on all DSM versions, since we add DSM 7 compatible folders for DSM 5, 6 and SRM 1.2.

Reply:
I don't know why I thought it was differing depending on DSM versions.

Will be updating it (this will even simplify the code load !)